service/src/directory.js

```diff
@@ -53,7 +56,7 @@ exports.getUsers = async (req) => {
 // update a user
 exports.updateUser = async (req, user, payload) => {
   try {
-    const response = await directoryClient.setObject({ object: payload });
+    const response = await directoryClient.setObject(fromJson(SetObjectRequestSchema, { object: payload }));
```

Comment:
At the very least it seems like we should have a `fromJson` which is imported from @aserto-node not from @bufbuild/protobuf.